pii_recognition/evaluation/pipeline_test.py

```diff
@@ -3,17 +3,21 @@
 from .pipeline import get_recogniser
 
 
-@patch("pii_recognition.evaluation.pipeline.registry")
-def test_get_recogniser(mock_regsitry):
-    class Simple:
-        pass
+class Simple:
+    pass
 
-    class Complex:
-        def __init__(self, a):
-            self.a = a
 
-    mock_regsitry.recogniser = {"Simple": Simple, "Complex": Complex}
+class Complex:
```

Comment:
Can we give more meaningful names other than just `Simple` and `Complex` pls?

Reply:
For sure! I've rename to `RegistryNoConfig` and `RegistryWithConfig` along with a line of comment to each renamed class. Let me know if it makes sense.

Reply:
makes much more sense now! 👍